app/models/views/user/statements.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module Views
+  module User
+    class Statements
+      include ActiveModel::Model
+      attr_accessor :overviews
+
+      MONTH_FORMAT = "%b %Y"
+      YEAR_FORMAT = "%b %e, %Y"
+
+      def initialize(publisher:, details_date: nil)
+        @statements = PublisherStatementGetter.new(publisher: publisher).perform
+        build_statements(publisher, details_date)
+      end
+
+      def as_json(*)
+        {
+          overviews: overviews,
+        }
+      end
+
+      def build_statements(publisher, details_date)
+        @overviews = []
```

Comment:
I think you can keep this as a local variable

Reply:
I don't think so, because this is what is being serialized and sent as json to the view